### DISCUSSION TITLE
Additional server support

### DISCUSSION THREAD

**Discussion by ippocratis** [savvasdalkitsis/uhuruphotos-android#201] [Ideas]
There are some great self hosted photo managment projects around
Nextcloud
Photoprism
Immich
Photoview
Just to name some
I can imagine the additional work to implement the additional api's and even different programming languages is a simple no go
but would that even be doable or even thinkable?

**Comment by savvasdalkitsis** [comment]
I have thought about it. It will require some significant work but it is definitely a good idea. One for after the first public release of course :)
